transforms/code/repo_level_ordering/ray/src/dpk_repo_level_order/internal/repo_grouper.py

```diff
@@ -62,14 +64,26 @@ def _write_parquet(self, table, repo_name):
         self.data_access.save_table(os.path.normpath(parquet_path), table)
 
     def _read_parquet_bulk(self, files):
-        tables = list(
+        def get_dataframes(path):
+            table, _ = self.data_access.get_table(path)
+            if table is not None:
+                return table.to_pandas()
+            else:
+                return None
+
+        tables_ = list(
             map(
-                lambda x: self.data_access.get_table(os.path.normpath(x))[0].to_pandas(),
+                lambda x: get_dataframes(os.path.normpath(x)),
```

Comment:
if lamda function failed to read one or more files, then function should record the failure. Currently, it ignores the failure. IMHO, it is critical error, it should flag the error & terminate.

Reply:
The recommendation is to have the transform() throw the exception to the runtime.  See https://github.com/IBM/data-prep-kit/blob/dev/data-processing-lib/doc/transform-exceptions.md